install_config/router/f5_router.adoc

```diff
@@ -26,6 +26,12 @@ like the
 xref:../../install_config/router/default_haproxy_router.adoc#install-config-router-default-haproxy[default
 HAProxy router].
 
+[IMPORTANT]
+====
+Support relationships between F5 and Red Hat provide a full scope of support for
+F5 integration, while Red Hat alone provides limited support for F5 integration.
+====
```

Comment:
We support the integration fully still. We do not support the F5 product itself. So this statement maybe a little misleading. Something on these lines maybe?

Support relationships between F5 and Red Hat provide full scope of support for F5 integration. F5 provides support for the BIG-IP LTM product. Both F5 and Red Hat jointly support the integration with OpenShift.